src/neo4j_graphrag/experimental/pipeline/pipeline.py

```diff
@@ -15,12 +15,12 @@
 from __future__ import annotations
 
 import asyncio
+import datetime
 import enum
 import logging
 import uuid
 import warnings
 from collections import defaultdict
-from datetime import datetime
```

Comment:
Why change the import?

Reply:
Because we need `datetime.timezone` below from the top level package. Could also add `from datetime import timezone`

Reply:
Ah, I missed that! Fair enough